File: v2/csharp/src/main/csharp/io/swagger/Model/Eventresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Eventresponse {
    

    
    public Event Event { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Eventresponse {\n");
      
      sb.Append("  Event: ").Append(Event).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/Eventresponse.java
package io.swagger.client.model;

import io.swagger.client.model.Event;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Eventresponse  {
  
  @SerializedName("event")
  private Event event = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Event getEvent() {
    return event;
  }
  public void setEvent(Event event) {
    this.event = event;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Eventresponse {\n");
    
    sb.append("  event: ").append(event).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
